test/test-utils/audio.ts

```diff
@@ -63,7 +63,7 @@ export const createTestPlayback = (): Playback => {
         liveData: new SimpleObservable<number[]>(),
         durationSeconds: 31415,
         timeSeconds: 3141,
-    } as PublicInterface<Playback> as Playback;
+    } as unknown as Playback;
```

Comment:
I used `PublicInterface` here so that typescript can tell me if I miss anything in my mock. Can we keep it?